app/views/hiring_positions/_hiring_position.html.erb

```diff
@@ -0,0 +1,18 @@
+<div id="<%= dom_id hiring_position %>">
+
+  <div class="card me-4 mt-2" style="width: 18rem;">
+    <% if hiring_position.hiring_image.attached? %>
+    <%= image_tag hiring_position.hiring_image, class: "card-img-top" %>
+    <% end %>
+    <div class="card-body">
+      <h5 class="card-title"><%= hiring_position.title %></h5>
+      <h6 class="card-subtitle mb-2 text-muted"><%= hiring_position.location %></h6>
+      <p class="card-text"> <%= hiring_position.details %> </p>
+
+      <%= link_to "Show Candidates", hiring_position_candidates_path(hiring_position), class: "btn btn-primary" %>
+      <%= link_to "Edit this hiring position", edit_hiring_position_path(hiring_position), class: "btn btn-success mt-2" %> 
+      <%= button_to "Destroy this hiring position", hiring_position, method: :delete, class: "btn btn-danger mt-2" %>
```

Comment:
change these messages also, message must be precise like Edit this will be enough